### PR TITLE
[CROSSORG-13] Update cross-org public docs to reflect launch of public beta

### PR DESCRIPTION
**Pull request by aaronherrmann**
### What does this PR do? What is the motivation?
[CROSSORG-13] Update cross-org public docs to reflect transition from private beta to public beta.

### Merge instructions
Hold for merge/release until public beta is released; projected release date is *8 July 2024*

- [ ] Please merge after reviewing

### Additional notes
No

### CODE REVIEW

**Review by dabickle — APPROVED** [review]

**Comment by hestonhoffman** [comment]
@aaronherrmann Hi! Looks like there are some merge conflicts in this one. Let us know if you need help with that

File: content/en/account_management/org_settings/cross_org_visibility.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
 ---
 title: Cross-Organization Visibility
 kind: documentation
-private: true
 is_beta: true
 ---
 
 {{< callout url="#" header="false" btn_hidden="true">}}
-  Cross-organization visibility is in <strong>private beta</strong> for customers with Enterprise plans. If you're interested in the feature, reach out to your Technical Account Manager or Customer Success Manager.
+  Note: Cross-organization visibility is in <strong>public beta</strong>. If you have any feedback, contact [Datadog support](https://docs.datadoghq.com/help/).
```
Comment (hestonhoffman) [inline content/en/account_management/org_settings/cross_org_visibility.md:8]:
```suggestion
  Note: Cross-organization visibility is in beta. If you have any feedback, contact [Datadog support](https://docs.datadoghq.com/help/).
```
We don't usually specify "public" in these.

File: content/en/account_management/org_settings/cross_org_visibility_api.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
 ---
 title: Cross-Organization Connections API
 kind: documentation
-private: true
 is_beta: true
 ---
 
 {{< callout url="#" header="false" btn_hidden="true">}}
-  Cross-organization visibility is in <strong>private beta</strong> for customers with Enterprise plans. If you're interested in the feature, reach out to your Technical Account Manager or Customer Success Manager.
+  Note: Cross-organization visibility is in <strong>public beta</strong>. If you have any feedback, contact [Datadog support](https://docs.datadoghq.com/help/).
```
Comment (hestonhoffman) [inline content/en/account_management/org_settings/cross_org_visibility_api.md:8]:
```suggestion
  Note: Cross-organization visibility is in beta. If you have any feedback, contact [Datadog support](https://docs.datadoghq.com/help/).
```

**Review by hestonhoffman — APPROVED** [review]

**Comment by hestonhoffman** [comment]
@aaronherrmann I'm going to go ahead and fix this one for you so we can get it merged. :shipit:

**Comment by hestonhoffman** [comment]
Done. Should be good to merge. Just let us know when it's ready to go!

File: content/en/account_management/org_settings/cross_org_visibility.md (hunk 1 above)
Comment (aaronherrmann) [reply content/en/account_management/org_settings/cross_org_visibility.md:7]: > We don't usually specify "public" in these.
Except for all the places where we do: https://www.google.com/search?q=site%3Adocs.datadoghq.com+%22public+beta%22

Accepting this one

**Comment by aaronherrmann** [comment]
Ready to merge and promote once approved, as public beta code has been pushed to production.

**Pull request merged**